packages/ssi-sdk-core/package.json

```diff
@@ -10,6 +10,7 @@
   "dependencies": {
     "typeorm": "^0.3.10",
     "@scure/base": "^1.1.1",
+    "@sphereon/did-auth-siop": "0.3.0-unstable.1",
```

Comment:
Why do we need to depend on did-auth-siop from core?

Reply:
reverted